source/tyr/tests/integration/datapost_test.py

```diff
@@ -6,14 +6,18 @@
 from tyr import app
 import logging
 
-@pytest.fixture(scope="module")
+@pytest.fixture
 def create_instance_fr():
     with app.app_context():
         instance = models.Instance('fr')
+        ac_instance = models.AutocompleteParameter(name='fr', street='OSM', address='BANO', poi='OSM', admin='OSM',
```

Comment:
I think it's a good practice not to have the same name for autocomplete instance and navitia instances, it's already confusing :stuck_out_tongue_winking_eye: